composer.json

```diff
@@ -6,8 +6,12 @@
     "homepage": "https://github.com/Flubit/merchant-api-client",
     "license": "MIT",
     "require": {
-        "php": ">=5.3.3",
-        "guzzle/guzzle": "~3.7"
+        "php": ">=5.4",
+        "guzzlehttp/guzzle": "~5.3"
```

Comment:
I'm thinking if we really want to set minimum PHP version to `5.4`. I can believe that a lot of developers don't have comfort of using version of PHP that they want. 

Is there something in version `5.3` of guzzle that we're using and is not available in version `3.9.3`? Version `3.9.3` required PHP 5.3.3
Just trying to justify use of newest guzzle :)